contrib/download-frozen-image-v2.sh

```diff
@@ -203,10 +203,10 @@ handle_single_manifest_v2() {
 
 	local manifestJsonEntry
 	manifestJsonEntry="$(
-		echo '{}' | jq --raw-output '. + {
+		echo '{}' | jq -rjc '. + {
 			Config: "'"$configFile"'",
 			RepoTags: ["'"${image#library\/}:$tag"'"],
-			Layers: '"$(echo '[]' | jq --raw-output ".$(for layerFile in "${layerFiles[@]}"; do echo " + [ \"$layerFile\" ]"; done)")"'
+			Layers: '"$(echo '[]' | jq -rjc ".$(for layerFile in "${layerFiles[@]}"; do echo " + [ \"$layerFile\" ]"; done)")"'
```

Comment:
I don't think I understand why we need `-j`, but I **don't** agree we should use `-c` (because not using it makes the output much easier to read).

I also think if we _do_ need `-j`, we should probably stick to the long form like we were using before (https://jqlang.github.io/jq/manual/v1.6/, `--join-output`, which I think implies `-r` although I have to admit I'm not terribly familiar with it because I've never needed that in all my [ab]use of `jq` so far :joy:).

Can you elaborate a bit on the problem you ran into that you are solving here with `-j`?

Reply:
I just retested and it doesn't need -j.  It definitely needs the -c for the inner-most jq statement.  When I was trying to pull docker images before, it was hanging up on the fact that the  inner-most "jq" statement had newline characters in it.  This is the error:  "jq: error: syntax error, unexpected INVALID_CHARACTER (Windows cmd shell quoting issues?) at <top-level>, line 1:
".  By adding the -c, it would make sure that no newlines were included and allow the outer "jq" statements to execute.  This is on windows git-bash instance with jq installed separately.  I guess I left the -j in there from previous attempts to get it working.  You can omit the -c on the outer-most jq statement (~line 439 for manifestJsonEntries list) to preserve readability, which I did.  Happy to have it be the long form (e.g. --compact-output).